frontend/src/helpers/isDockerDesktop.ts

```diff
@@ -0,0 +1,18 @@
+declare global {
```

Comment:
Can you please add comment for this? Thanks

Reply:
done